doc/manual/src/installation/prerequisites-source.md

```diff
@@ -35,7 +35,10 @@
   - The [Boehm garbage collector](http://www.hboehm.info/gc/) to reduce
     the evaluator’s memory consumption (optional). To enable it, install
     `pkgconfig` and the Boehm garbage collector, and pass the flag
-    `--enable-gc` to `configure`.
+    `--enable-gc` to `configure`. Nix relies on a patch to support the
+    necessary interactions between coroutines and the collector; please
+    apply `boehmgc-coroutine-sp-fallback.diff` to your Boehm garbage collector
```

Comment:
Please add a link to the file on GitHub so people can find out what specifically you mean.

This is the same issue as https://github.com/NixOS/nix/pull/7412#discussion_r1042454626.